identify/interpreters.py

```diff
@@ -6,6 +6,7 @@
     'bash': {'shell', 'bash'},
     'csh': {'shell', 'csh'},
     'dash': {'shell', 'dash'},
+    'nix-shell': {'shell', 'bash'},
```

Comment:
This is suitable as a first-order approximation, because most of the time it will be `-i bash` indeed.
It'd be better to write a function to parse the consecutive `#!nix-shell` lines to find the `-i` value. You could call the new function from around here https://github.com/chriskuehl/identify/blob/62e98d7bee975f686bab8257c4fe95ff0ff7ab80/identify/identify.py#L162

Reply:
Thanks! In other words: please.merge and follow up with another PR. :wink: